src/util/pg.py

```diff
@@ -290,8 +290,23 @@ def create_column(cr, table, column, definition, **kwargs):
     # Manual PEP 3102
     no_def = object()
     default = kwargs.pop("default", no_def)
+    fk = kwargs.pop("fk", "")
     if kwargs:
         raise TypeError("create_column() got an unexpected keyword argument %r" % kwargs.popitem()[0])
+
+    if fk:
+        if isinstance(fk, str):
+            fk = (fk, "id", "SET NULL")
+        if not (isinstance(fk, tuple) and len(fk) == 3 and all(isinstance(e, str) for e in fk)):
+            raise ValueError("`fk` is expected to be a table name or a 3-tuple (table_name, column, on_delete_action)")
+        if fk[2].upper() not in ("SET NULL", "CASCADE", "RESTRICT"):
```

Comment:
Extract those values in a constant, to be consistent with " create_fk" definition below.